spinnaker_camera_driver/include/spinnaker_camera_driver/SpinnakerCamera.h

```diff
@@ -189,6 +196,8 @@ class SpinnakerCamera
   /// If true, camera is currently running in color mode, otherwise camera is running in mono mode
   bool isColor_;
 
+  int ptp_time_offset_secs_;
```

Comment:
i don't see where some of the other vars are init'd (ie didn't ctrl-f to ctor init list), but these members should be init'd somewhere - make sure this one is, at least, so we don't start with a random ptp offset.